pkg/lightning/errormanager/errormanager.go

```diff
@@ -269,6 +281,14 @@ func (em *ErrorManager) Init(ctx context.Context) error {
 		}
 	}
 
+	if em.conflictV1Enabled && em.conflictV2Enabled {
+		sql := [2]string{"create conflict view", createConflictView}
```

Comment:
no need to use this variable? Directly `exec.Exec(ctx, "create conflict view", common.SprintfWithIdentifiers(createConflictView, em.schema, em.schema, em.schema))`